File: iPad/VerBank-CSTSv3-IOSClientApi/CSTS3Comm/ipop/OP_Ctrl5106.m
//
//  OP_Ctrl5106.m
//  VerBank-IOSClientStation
//
//  Created by Allone on 15/10/21.
//  Copyright © 2015年 zhanglei. All rights reserved.
//

#import "OP_Ctrl5106.h"

static NSString *jsonId               = @"OP_Ctrl5106";

static NSString *returnCode         = @"1";
static NSString *state              = @"2";

@implementation OP_Ctrl5106

-(id)init {
    if (self = [super init]) {
        [super setEntry:@"jsonId" entry:jsonId];
    }
    return self;
}

jsonImplementionInt     (ReturnCode,    returnCode)
jsonImplementionInt     (State,         state)

@end

File: WeChatService/VerBank-Ctrl/src/allone/MTP/VerBank01/Ctrl/comm/ipop/OP_Ctrl5106.java
package allone.MTP.VerBank01.Ctrl.comm.ipop;

/**
 * 
 * @Project: VerBank-Ctrl 
 * @Title: OP_Ctrl5106.java 
 * @Package allone.MTP.VerBank01.Ctrl.comm.ipop 
 * @Description: 取得憑證　(GetCertState)
 * @author wyb wang.yubo@allone.cn 
 * @date 2015年10月15日 下午4:44:16 
 * @Copyright: 2015 www.allone.cn Inc. All rights reserved. 
 * @version V3.0.0   
 */
public class OP_Ctrl5106 extends CtrlServerOPFather {

	private static final long serialVersionUID = -2415371148750299558L;

	private int returnCode;

	private int state;

	public OP_Ctrl5106(CtrlServerIPFather ip) {
		super(ip);
	}

	public int getReturnCode() {
		return returnCode;
	}

	public void setReturnCode(int returnCode) {
		this.returnCode = returnCode;
	}

	public int getState() {
		return state;
	}

	public void setState(int state) {
		this.state = state;
	}

}
